drivers/windows/dir_access_windows.cpp

```diff
@@ -165,8 +165,11 @@ Error DirAccessWindows::make_dir(String p_dir) {
 	bool success;
 	int err;
 
-	p_dir = "\\\\?\\" + p_dir; //done according to
-	// https://msdn.microsoft.com/en-us/library/windows/desktop/aa363855(v=vs.85).aspx
+	if (!p_dir.begins_with("\\\\")) {
```

Comment:
Maybe having a `is_network_share_path()` would be clearer than putting `p_dir.begins_with("\\\\")` in multiple places?

Reply:
This makes sense, and also can be used by the file dialog UI in some way.